src/Microsoft.ML.Transforms/GcnTransform.cs

```diff
@@ -445,16 +446,16 @@ public Mapper(LpNormalizingTransformer parent, Schema inputSchema)
                 }
             }
 
-            protected override Schema.Column[] GetOutputColumnsCore()
+            protected override Schema.DetachedColumn[] GetOutputColumnsCore()
             {
-                var result = new Schema.Column[_parent.ColumnPairs.Length];
+                var result = new Schema.DetachedColumn[_parent.ColumnPairs.Length];
                 for (int i = 0; i < _parent.ColumnPairs.Length; i++)
                 {
-                    var builder = new Schema.Metadata.Builder();
+                    var builder = new MetadataBuilder();
                     builder.Add(InputSchema[ColMapNewToOld[i]].Metadata, name => name == MetadataUtils.Kinds.SlotNames);
                     ValueGetter<bool> getter = (ref bool dst) => dst = true;
-                    builder.Add(new Schema.Column(MetadataUtils.Kinds.IsNormalized, BoolType.Instance, null), getter);
-                    result[i] = new Schema.Column(_parent.ColumnPairs[i].output, _types[i], builder.GetMetadata());
+                    builder.Add(MetadataUtils.Kinds.IsNormalized, BoolType.Instance, getter);
```

Comment:
>builder.Add(MetadataUtils.Kinds.IsNormalized, BoolType.Instance, getter); [](start = 20, length = 73)

this one is kinda frequent one, do you want to add it as proper method?

Reply:
Not in this PR.

---
In reply to: [235493173](https://github.com/dotnet/machinelearning/pull/1680#discussion_r235493173) [](ancestors = 235493173)